srcpkgs/luakit/template

```diff
@@ -1,21 +1,35 @@
 # Template file for 'luakit'
 pkgname=luakit
-version=2017.08.10
+version=2.1
 revision=1
-nocross=yes
-hostmakedepends="pkg-config"
+make_build_args="DEVELOPMENT_PATHS=0 LUA_BIN_NAME=luajit USE_LUAJIT=1 \
```

Comment:
Setting build_style appropriately should fix that

Reply:
Funny thing is, if you use gnu-makefile build style, it can’t find headers to build. If I leave it how it was it doesn’t complain and also runs when I tested it

Reply:
Reverting back to how it was originally, as using build_style causes linker issues when I resolve all the header missing errors. I tested it with the original setup and it built and ran fine